django/src/rdwatch/views/region.py

```diff
@@ -1,13 +1,39 @@
-from rest_framework import viewsets
-from rest_framework.pagination import PageNumberPagination
+import iso3166
+from ninja import Schema
+from ninja.pagination import RouterPaginated
+
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
 
 from rdwatch.models import Region
-from rdwatch.serializers import RegionSerializer
```

Comment:
I didn't remove this serializer yet since it's used elsewhere. But eventually, `RegionSerializer` will be replaced by the new `RegionSchema`